server/views/applications/view.njk

```diff
@@ -96,10 +96,10 @@
         {% if key.kind === 'api-client#web' %}
         <div class="govuk-summary-list__row">
           <dt class="govuk-summary-list__key">
-            Client Secret
+            <label for="{{ key.id+"-secret" }}">Client Secret</label>
```

Comment:
Any particular reason you changed this?

Reply:
There was a html validator error since technically there is an input field without a matching label, even if its not being used and is disabled

Reply:
Is it safe to expose the key ID here?

Reply:
We could think about another way of doing it - the problem is that they keyId is exposed like this in the input as well and was also exposed in the webkit security method before that. We need to give each client secret a unique id so that the corresponding show/hide only changes that one client secret